ivy/functional/ivy/experimental/layers.py

```diff
@@ -621,6 +621,77 @@ def dropout1d(
     )
 
 
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+def separable_conv2d(
+    x: Union[ivy.Array, ivy.NativeArray],
+    depthwise_filter: Union[int, Tuple[int], Tuple[int, int, int]],
+    pointwise_filter: Union[int, Tuple[int], Tuple[int, int, int]],
```

Comment:
As you can see the function requires two filters, but at test time you are generating only one kernel. You should be generating two kernels/filters.